<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctkDICOMQueryRetrieveWidget</class>
 <widget class="QWidget" name="ctkDICOMQueryRetrieveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>501</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>DICOM Query/Retrieve</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="QueryCollapsibleButton">
     <property name="text">
      <string>Query</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="ctkDICOMServerNodeWidget" name="ServerNodeWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>200</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Search Options</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="ctkDICOMQueryWidget" name="QueryWidget" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="QueryButton">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Query</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="RetrieveCollapsibleButton">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Retrieve</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="ctkDICOMTableManager" name="dicomTableManager"/>
      </item>
      <item>
       <widget class="QPushButton" name="RetrieveButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Retrieve</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="CancelButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkDICOMTableManager</class>
   <extends>QWidget</extends>
   <header>ctkDICOMTableManager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDICOMQueryWidget</class>
   <extends>QWidget</extends>
   <header>ctkDICOMQueryWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDICOMServerNodeWidget</class>
   <extends>QWidget</extends>
   <header>ctkDICOMServerNodeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
